src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -156,6 +170,8 @@ public Xcode(string target)
             dllMap.AppendLine($"    mono_dllmap_insert (NULL, \"{aFileName}\", NULL, \"__Internal\", NULL);");
         }
 
+        dllMap.AppendLine($"    mono_dllmap_insert (NULL, \"System.Globalization.Native\", NULL, \"__Internal\", NULL);");
```

Comment:
I did not realize we're doing this in `AppleAppBuilder`. That's a little unfortunate, but a problem for another day I guess.

Reply:
Wasn't the point of adding qcall support so that we could completely get rid of this stuff?